### ISSUE TITLE
after setting persistent_context=True,After restarting the browser, the same user_data_path was passed in, but the login was not successful

### ISSUE THREAD

**Issue by satanmp** [daijro/camoufox#69]
### Describe the bug:

I use persistent_context=True,user_data_dir="D:/userdata_test',After I successfully logged in,restarting the browser, the same user_data_path was passed in, but the login was not successful

### Screenshots:

![图片](https://github.com/user-attachments/assets/0171223a-6091-4477-b2ba-93d2cd74bb03)

### To Reproduce:

                with Camoufox(persistent_context=True,
                              user_data_dir=user_data_path,
                              # geoip=public_ip,
                              headless=False,
                              locale=['zh-CN', 'zh'],
                              i_know_what_im_doing=True,
                              # proxy={"server": proxy_ip},
                              # config={
                              #     'webrtc:ipv4': public_ip,
                              #     'geolocation:latitude': latitude,
                              #     'geolocation:longitude': longitude,
                              # },
                              humanize=True,
                              slow_mo=30, block_images=True, ) as browser:
                    page = browser.new_page()
                    page.goto("https://www.zhihu.com/")

### Version:

beta.14

**Comment by daijro** [comment]
Hello, thanks for reporting this. This is caused by Camoufox clearing data on shutdown:

```js
// Sanitize (from the SanitizeOnShutdown pref)
// https://searchfox.org/mozilla-central/source/browser/components/enterprisepolicies/Policies.sys.mjs#2088

defaultPref("privacy.clearOnShutdown.cache", true);
defaultPref("privacy.clearOnShutdown.cookies", true);
defaultPref("privacy.clearOnShutdown.downloads", true);
defaultPref("privacy.clearOnShutdown.formdata", true); // Form & Search History
defaultPref("privacy.clearOnShutdown.history", true); // Browsing & Download History
defaultPref("privacy.clearOnShutdown.offlineApps", true);
defaultPref("privacy.clearOnShutdown.openWindows", true); // Opened windows will not be restored
defaultPref("privacy.clearOnShutdown.sessions", true); // Active Logins
defaultPref("privacy.clearOnShutdown.siteSettings", true); // Site Preference

// v2
defaultPref("privacy.clearOnShutdown_v2.cache", true);
defaultPref("privacy.clearOnShutdown_v2.cookiesAndStorage", true);
defaultPref("privacy.clearOnShutdown_v2.historyFormDataAndDownloads", true);
defaultPref("privacy.clearOnShutdown_v2.siteSettings", true);

defaultPref("privacy.sanitize.sanitizeOnShutdown", true);
```

Since Playwright handles this on its own, I will remove these preferences in the latest update. A temporary workaround could be to disable these.

**Comment by daijro** [comment]
Fixed in v132.0-beta.15.
